db/migrations/0017.remove_user_table.py

```diff
@@ -0,0 +1,16 @@
+#
+# file: migrations/0017.remove_user_table.py
```

Comment:
This is `17` in anticipation of PR #151 being merged.

Reply:
I feel like it would have been nice if we could have separated out these migrations that don't have any dependencies to their own directories.

Like we could have had 
mivideo/migrations
online_meetings/migrations
course_inventory/migrations

Then the migrations with no prior dependencies wouldn't wait on others. I'm working on one right now that I'm just calling 18_ to stay in line. Maybe something to fix if we ever have time in the future. ;)

Reply:
Mmm, I see your point. I think I nixed this idea in the past because I thought it would unnecessarily complicate the codebase. Having a couple extra commits to change numbers when you're working at the same time as someone else felt like a small price to pay for less directories and keeping `db_creator.py` simpler. There probably is a way to do it though and maybe it wouldn't be too disruptive.

Reply:
Yeah, I know Django ORM does it by having every class extend migrations.Migration.

But we're doing it manually in the db_creator -> migrate -> read_migrations. 

We'd either have to hardcode all the paths (probably not too hard) to all the migrations or have some type of similar discovery process. 

Still I don't think it's a high priority.